hipify_cli.py

```diff
@@ -69,8 +69,9 @@ def main():
         output_directory=out_dir,
         includes=args.includes,
         ignores=args.ignores,
-        header_include_dirs=args.header_include_dirs,
-        is_pytorch_extension=True)
+        header_include_dirs=args.header_include_dirs.strip("[]").split(";"),
```

Comment:
Please confirm if this strip and split, will not cause any side effects for other scenarios.

Reply:
In `cmake/Hipify.cmake' we have `--header-include-dirs [${HIPIFY_HEADER_INCLUDE_DIR}]`. The square brackets always exist. I tested empty and one-element `${header_include_dir}` in `CMakeList.txt`, it doesn't fail because of this change.

Reply:
@pruthvistony  @liligwu After merging manually with current master , when using config_json option I see an error because of this strip : 
tried with this json:
```
{
    "project_directory":".",
    "header_include_dirs": "['include']",
    "includes":["src/*","tests/*", "pytorch_binding/*", "tensorflow_binding/*", "include/*" ],
    "dump_dict_file":"hipdump.txt"
}
```
```
project_directory : /var/lib/jenkins/warp-transducer/.  output_directory:  /var/lib/jenkins/warp-transducer/.  includes:  ['src/*', 'tests/*', 'pytorch_binding/*', 'tenslude_dirs:  ['include']
Traceback (most recent call last):
  File "/var/lib/jenkins/warp-transducer/third_party/hipify_torch/hipify_cli.py", line 124, in <module>
    main()
  File "/var/lib/jenkins/warp-transducer/third_party/hipify_torch/hipify_cli.py", line 113, in main
    header_include_dirs=args.header_include_dirs.strip("[]").split(";"),
AttributeError: 'list' object has no attribute 'strip'
CMake Error at third_party/hipify_torch/cmake/Hipify.cmake:109 (message):
  Failed to hipify files!
Call Stack (most recent call first):
  CMakeLists.txt:131 (hipify)


-- Configuring incomplete, errors occurred!
```

Reply:
@bmedishe, I pushed a new commit to the cmake_enable_include_dir branch that would fix this issue. Would you please try again? Thank you.

Reply:
@liligwu , Thank you Li , because of this PR , I was able to [fix a bug in hipify_cli.py ](https://github.com/ROCmSoftwarePlatform/hipify_torch/pull/23)

Reply:
The code looks OK now. Please comment if any issue is found.